latest/ug/nodes/hybrid-nodes-nodeadm.adoc

```diff
@@ -325,6 +325,19 @@ nodeadm uninstall [flags]
 `--help`
 |FALSE
 |Displays help message with available flag, subcommand and positional value parameters.	
+
+|`-f`,
+
+`--force`
+|FALSE
+|Force delete additional directories that might contain leftovers from the node process.
+
+*WARNING*
+
+- This will delete all contents in default Kubernetes and CNI directories (/var/lib/cni, /etc/cni/net.d, etc). Do not use this flag if you store your own data in these locations.
```

Comment:
```suggestion
This will delete all contents in default Kubernetes and CNI directories (/var/lib/cni, /etc/cni/net.d, etc). Do not use this flag if you store your own data in these locations.
```